src/js/components/ServicesTable.js

```diff
@@ -118,23 +115,25 @@ var ServicesTable = React.createClass({
       );
     }
 
-    if (service instanceof Framework) {
-      // Get framework image/icon
-      itemImage = (
-        <span
-          className="icon icon-small icon-image-container icon-app-container icon-margin-right">
-          <img src={service.getImages()['icon-small']}/>
-        </span>
-      );
-    }
+    // Get service image/icon
+    return (
+      <span
+        className="icon icon-small icon-image-container icon-app-container icon-margin-right">
+        <img src={service.getImages()['icon-small']}/>
+      </span>
+    );
+  },
```

Comment:
We test for everything `ServiceTree` here: https://github.com/dcos/dcos-ui/pull/709/files#diff-a4468ae3d6f9c66e921365d98b241156R106
The only thing that can be in this list is `ServiceTree` or `Service`